src/network/address.rs

```diff
@@ -81,7 +84,7 @@ impl Encodable for Address {
     ) -> Result<usize, io::Error> {
         let len = self.services.consensus_encode(&mut s)?
             + addr_to_be(self.address).consensus_encode(&mut s)?
-            + self.port.to_be().consensus_encode(s)?;
+            + self.port.swap_bytes().consensus_encode(s)?;
```

Comment:
Can you add a comment as to why swap_bytes here, maybe pasting andrew's review comment?

Reply:
Added comments, also suggesting to switch to other code when MSRV allow that